tools/backups2datalad.py

```diff
@@ -428,6 +428,9 @@ def get_dandisets(
     def describe_dandiset(self, dandiset: RemoteDandiset) -> str:
         metadata = dandiset.get_raw_metadata()
         desc = dandiset.version.name
+        versions = sum(1 for v in dandiset.get_versions() if v.identifier != "draft")
+        if versions:
```

Comment:
Please add contact first to desc then I'm a similar fashion (now done at the end)

Reply:
Done.